README.md

```diff
@@ -1,12 +1,34 @@
-# Nym VPN client
+Hello and welcome to the NymVPN GitHub page! For more information about NymVPN and to request beta access credentials, visit https://nymvpn.com/en.
 
-VPN client that uses the [Nym mixnet](https://nymtech.net). 
+# NymVPN client
 
-- 5-hops (incl entry and exit gateways), 
-- optional: 2-hop straight from entry to exit gateway.
-- optional: tunnel the connection to the entry gateway through wireguard
+The NymVPN client is a VPN-like app built on [Nym's signature mixnet](https://nymtech.net), offering the following [features](https://nymvpn.com/en/features):
+- Anonymous 5-hop mixnet mode
+- Fast 2-hop WireGuard-based decentralized VPN mode
+- Private credentials using zk-nyms (zero-knowledge proofs)
+
+NymVPN is available on all major platforms incl. [Android](https://nymvpn.com/en/download/android), [iOS](https://nymvpn.com/en/download/ios), [Linux](https://nymvpn.com/en/download/linux), [macOS](https://nymvpn.com/en/download/macos) and [Windows](https://nymvpn.com/en/download/windows).
+
+NymVPN relies on [Mullvad open source libraries](https://github.com/mullvad/mullvadvpn-app/) to handle setting up local routing and wrapping wireguard-go.
```

Comment:
@pronebird is this still relevant?

Reply:
Yes, that's correct with exception of WireGuard on mobile.